WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostPreviewFragment.java

```diff
@@ -32,7 +37,7 @@ public static EditPostPreviewFragment newInstance(SiteModel site, PostModel post
         EditPostPreviewFragment fragment = new EditPostPreviewFragment();
         Bundle bundle = new Bundle();
         bundle.putSerializable(WordPress.SITE, site);
-        bundle.putSerializable(EditPostActivity.EXTRA_POST, post);
+        bundle.putInt(EditPostActivity.EXTRA_POST_LOCAL_ID, post.getId());
```

Comment:
I agree and I disagree. We use `putSerializable` because we want to avoid passing ids, especially we want to avoid confusion with remote and local ids.